iset.mm

```diff
@@ -93987,6 +93987,22 @@ gives an ordered pair whose first element is the index (translated
       ax-mp mpbir ) DEBFZDGHIZBFZUBJAGBJKCLMEUANTUBOPEUADBQRS $.
   $}
 
+  ${
+    $d C x $.  $d ph x $.
+    frec2uzled.1 $e |- ( ph -> C e. ZZ ) $.
+    frec2uzled.2 $e |- G = frec ( ( x e. ZZ |-> ( x + 1 ) ) , C ) $.
+    frec2uzled.a $e |- ( ph -> A e. _om ) $.
+    frec2uzled.b $e |- ( ph -> B e. _om ) $.
+    $( The mapping ` G ` (see ~ frec2uz0d ) preserves order.  (Contributed by
+       Jim Kingdon, 24-Feb-2022.) $)
+    frec2uzled $p |- ( ph -> ( A C_ B <-> ( G ` A ) <_ ( G ` B ) ) ) $=
+      ( wcel wceq wo cfv wbr com wb syl2anc cz frec2uzzd clt wss cle wf1o f1of1
```

Comment:
I just had a quick look at http://us2.metamath.org/ileuni/frec2uz0d.html and that section.
Is it worth to define, either with an $a-stmt or in $e-stmts,
```
G = ( x e. CC |-> frec ( ( y e. CC |-> ( y + 1 ) ) , x ) )
```
(maybe replacing CC with ZZ or NN0) ?

Reply:
Hmm, there are a few ways to answer that:

1. What we do now is mostly as the above comment suggests, except that the comment has an extra function, which I think would just need to be unwrapped. Pretty much everywhere this is used, `C` is a fixed value (generally `0`) so needing to apply http://us.metamath.org/ileuni/fvmpt.html (or whatever we use to remove the extra function) seems unnecessary.
2. I'm just following what set.mm does here (in http://us2.metamath.org/mpeuni/om2uz0i.html and following theorems), I didn't think too hard about whether there is a better way. The correspondence between set.mm and iset.mm can be seen by searching for om2uz0i at http://us.metamath.org/ileuni/mmil.html
3. I suppose `CC` would work; I don't really remember why I chose `ZZ` rather than `CC` on on the one hand or ``( ZZ>= ` C )`` on the other. Of course it would be work to change now.